src/app/highlight-cards-resolver.ts

```diff
@@ -3,12 +3,20 @@ import {ActivatedRouteSnapshot, ResolveFn} from '@angular/router';
 import {environment} from "../environments/environment";
 import {inject} from "@angular/core";
 import {CampaignService} from "./campaign.service";
+import {catchError} from "rxjs/operators";
+import {of} from "rxjs";
 
 export const highlightCardsResolver: ResolveFn<readonly HighlightCard[] | null> = (route: ActivatedRouteSnapshot) => {
   const campaignService = inject(CampaignService);
 
   if (route.queryParams.hasOwnProperty('highlightAPIEnabled') && environment.environmentId !== 'production') {
-    return campaignService.getHomePageHighlightCards();
+    return campaignService.getHomePageHighlightCards().pipe(
+      // If the HighlightCards API has any error we still want to show the rest of the homepage, so we catch the error
+      catchError(error => {
+        console.error("Error fetching hompepage highlight cards", error);
+        return of([]);
```

Comment:
If we wanted instead of an empty list we of highlight cards we could put a card here apologising for the error and saying we'll hope to fix it soon. Not sure if that's better.